test/fixtures/full/base/outputs.tf

```diff
@@ -72,3 +72,14 @@ output "region" {
   value       = local.location
   description = "Subnetworks region created for bindings."
 }
+
+output "fixture_project_id" {
```

Comment:
An output which forwards an input is an anti-pattern; the input should be used directly in the calling module.